fighters/jack/src/acmd/smashes.rs

```diff
@@ -75,34 +73,27 @@ unsafe fn jack_attack_hi4_game(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
     frame(lua_state, 2.0);
-    if is_excute(fighter) {
-        FT_MOTION_RATE(fighter, 0.667);
-    }
+    FT_MOTION_RATE(fighter, 0.667);
     frame(lua_state, 7.0);
+    FT_MOTION_RATE(fighter, 1.000);
     if is_excute(fighter) {
-        FT_MOTION_RATE(fighter, 1.000);
         WorkModule::on_flag(boma, *FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD);
     }
     frame(lua_state, 8.0);
-    if is_excute(fighter) {
-        FT_MOTION_RATE(fighter, 0.667);
-    }
+    FT_MOTION_RATE(fighter, 0.667);
     frame(lua_state, 12.0);
+    FT_MOTION_RATE(fighter, 1.0);
     if is_excute(fighter) {
-        FT_MOTION_RATE(fighter, 1.000);
```

Comment:
lets change this to the standard `FT_MOTION_RATE` format `FT_MOTION_RATE(fighter, C / (B - A))`